### DISCUSSION TITLE
example for bulk insert/ batch insert to database

### DISCUSSION THREAD

**Discussion by divyanshurai07** [apache/camel-karavan#738] [Q&A] [answered]
hi , please give examples on how to use bulk insert or batch insert with databases , have tried the postgres SQL sink kamelet.

is there a component/kamelet available for bulk insert to databases(preferably oracle DB or postgres DB).

**Comment by oscerd** [comment, marked as answer]
No, there is not out of the box from Kamelets catalog.
